x-pack/plugins/secops/public/components/timeline/data_providers/mock/mock_data_providers.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as React from 'react';
+import { DataProvider } from '../data_provider';
+import { FacetText } from '../facet_text';
+
+interface NameToEventCount<TValue> {
+  [name: string]: TValue;
+}
+
+/**
+ * A map of mock data provider name to a count of events for
+ * that mock data provider
+ */
+const mockSourceNameToEventCount: NameToEventCount<number> = {
+  'Provider 1': 64,
+  'Provider 2': 158,
+  'Provider 3': 381,
+  'Provider 4': 237,
+  'Provider 5': 310,
+  'Provider 6': 1052,
+  'Provider 7': 538,
+  'Provider 8': 429,
+  'Provider 9': 708,
+};
+
+/** Returns a collection of mock data provider names */
+export const mockDataProviderNames = (): string[] => Object.keys(mockSourceNameToEventCount);
+
+/** Returns a count of the events for a mock data provider */
+export const getEventCount = (dataProviderName: string): number =>
+  mockSourceNameToEventCount[dataProviderName] || 0;
```

Comment:
I know this is just a mocks data provider, but this feels a bit odd that it returns `0` if it can't find anything. Feels a bit unsafe or just makes me question when/why you're returning `0` instead of being more intentful.